person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permision: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permision = parent_permision
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    if of_age? || parent_permision
+      true
+    else
+      false
+    end
+  end
```

Comment:
- [ ] in this setup the method `can_use_services?` is also a private method please make sure this is a public method.